OneCommunityGlobal/HGNRest
# Ivy-InformationController-Unit Test-Catch Err

# Description
Please include the exact bug/functionality description and a summary of the changes/ related issues. Please also include any other relevant motivation and context:
Fixes # (bug list priority high/medium/low x.y.z)
Or Implements # (WBS) 

## Related PRS (if any):
This frontend PR is related to the #XXX backend PR.
To test this backend PR you need to checkout the #XXX frontend PR.
…

## Main changes explained:
- Add InformationController.spec.js
- modify Information.js
- add getInformation.md
- add addInformation.md
- add updateInformation.md
- add deleteInformation.md


## How to test:
check into current branch
do npm install
run npm run test -- informationController.spec.js
verify all tests pass
<img width="540" alt="Screenshot 2024-04-29 at 11 30 07 AM" src="https://github.com/OneCommunityGlobal/HGNRest/assets/102696148/693edbea-85e6-4906-a9da-e60400a173c7">

## Screenshots or videos of changes:
